### PR TITLE
Skip persistently flaky PL test to debug

### PR DESCRIPTION
**Pull request by bencodeorg**
We've got a test that has been persistently flaky for a few weeks. Temporarily skipping so that we can work on a fix without blocking the DOTD.

### CODE REVIEW

File: dashboard/test/ui/features/plc/pd/dashboard_view.feature [hunk 1]
```diff
@@ -3,16 +3,6 @@
 
 Feature: Basic appearance for Facilitator Survey UI
 
-Scenario: Facilitator View of dashboard is as expected
```
Comment (molly-moen) [inline dashboard/test/ui/features/plc/pd/dashboard_view.feature:6]: do we have a way to temporarily disable this instead?

Comment (islemaster) [reply dashboard/test/ui/features/plc/pd/dashboard_view.feature:6]: We do!  Add a `@skip` annotation on the line before the scenario, like this:

https://github.com/code-dot-org/code-dot-org/blob/d8148e6f1bc583cf489a9c041d4466be73cb4db0/dashboard/test/ui/features/foundations/footer.feature#L86-L88

Comment (bencodeorg) [reply dashboard/test/ui/features/plc/pd/dashboard_view.feature:6]: Ah, I see -- so we can keep running this as a UI test but skip the eyes part? I see we have `@eyes` at the top of this test file, would putting `@eyes @skip` in front of the test supersede the `@eyes` at the top of the file?

Any idea where can I find docs on how this works? I'm looking through runner.rb and cucumber docs, but struggling to understand how @ is used? Anyone up to give me a quick primer?

Comment (islemaster) [reply dashboard/test/ui/features/plc/pd/dashboard_view.feature:6]: Sorry, I gave a poor example.  The `@eyes` and `@skip` tags operate independently; this is an eyes test, and entirely skipped.

[Here is Cucumber's documentation on tags.](https://cucumber.io/docs/cucumber/api/#tags) Tags are used to select a set of tests for Cucumber to run; but how that interacts with our test runner is dependent on our own implementation.

Here's where we _omit_ tests tagged `@skip` when running UI/Eyes tests:

https://github.com/code-dot-org/code-dot-org/blob/8008ec6aa13d78ea10b477ed930232c57f8fca68/dashboard/test/ui/runner.rb#L619-L621

Here's where we _select_ tests tagged `@eyes` if we are running eyes tests on any browser that is not mobile or Internet Explorer:

https://github.com/code-dot-org/code-dot-org/blob/8008ec6aa13d78ea10b477ed930232c57f8fca68/dashboard/test/ui/runner.rb#L626-L637

Comment (bencodeorg) [reply dashboard/test/ui/features/plc/pd/dashboard_view.feature:6]: Got it, thank you! Definitely thought `@eyes` and `@skip` were coupled, makes much more sense now! Updated.

So there's not actually a way to decouple eyes and UI tests? Sorry if I'm not using the right words, by "UI" I mean tests that check elements get rendered as described in these cucumber steps, and by "eyes" I mean pixel-level checks that everything is in the right place, which gets run in a much more limited set of environments by default (this is where we're seeing failures, yeah?)

**Review by molly-moen — APPROVED** [review]
🎉

File: dashboard/test/ui/features/plc/pd/dashboard_view.feature (hunk 1 above)
Comment (bencodeorg) [reply dashboard/test/ui/features/plc/pd/dashboard_view.feature:6]: Chatted with @islemaster offline -- eyes are subset of UI tests, this change will fully deactivate this test.

**Pull request merged**